src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/VSMachineWideSettings.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.ComponentModel.Composition;
+using System.Threading.Tasks;
```

Comment:
why are you adding this using?
https://github.com/Microsoft/VSProjectSystem/blob/master/doc/overview/cookbook.md makes me believe that taskscheduler is from Microsoft.VisualStudio.Threading???

Reply:
otherwise, looks good.

Reply:
[There's no TaskScheduler class in Microsoft.VisualStudio.Threading](https://www.fuget.org/packages/Microsoft.VisualStudio.Threading/). There's a `using Microsoft.VisualStudio.Threading` on line 8, so if there was a TaskScheduler in both System.Threading.Tasks and Microsofot.VisualStudio.Threading, then line 37 wouldn't compile with an ambiguous reference error.